irohad/main/iroha_conf_loader.hpp

```diff
@@ -20,55 +20,97 @@
 
 #include <fstream>
 #include <string>
+
 #include <rapidjson/rapidjson.h>
-#include <rapidjson/istreamwrapper.h>
+
 #include "common/assert_config.hpp"
+#include "common/types.hpp"
+#include "model/converters/json_common.hpp"
 
 namespace config_members {
-  const char* BlockStorePath = "block_store_path";
-  const char* ToriiPort = "torii_port";  // TODO: Needs AddPeer.
-  const char* KeyPairPath = "key_pair_path";
-  const char* PgOpt = "pg_opt";
-  const char* RedisHost = "redis_host";
-  const char* RedisPort = "redis_port";
+  auto BlockStorePath = "IROHA_BLOCK_STORE_PATH";
```

Comment:
`const`?

And for others below

Reply:
String literals are deduced as arrays of const characters.